doc/nrf/includes/build_and_run_nrf9160.txt

```diff
@@ -2,5 +2,6 @@ This sample can be found under |sample path| in the |NCS| folder structure.
 
 The sample is built as a non-secure firmware image for the nrf9160dk_nrf9160ns build target.
 Because of this, it automatically includes the :ref:`secure_partition_manager`.
+You can also configure it to use :ref:`TF-M <ug_tfm>` instead of SPM.
```

Comment:
This is boilerplate text that will be used in all 9160 samples.

Reply:
do you need to explain _how_ this could be done?

Reply:
We have this described in the user guide here -> http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_tfm.html#building

Do you think it is sufficient for the user?

Reply:
I think it is - i wonder if you could add a phrase like, "refer to the tf-m user guide for more details...."

Reply:
This ref to ug_tfm leads exactly to the user guide.

Reply:
Fine, I was jsut wondering if we should be more "verbose" :)